### ISSUE TITLE
Visiting click-that-hood.com throws a strong warning regarding an invalid SSL certificate

### ISSUE THREAD

**Issue by mwichary** [codeforgermany/click_that_hood#55]
When you visit https://click-that-hood.com:

You attempted to reach click-that-hood.com, but instead you actually reached a server identifying itself as *.heroku.com. This may be caused by a misconfiguration on the server or by something more serious. An attacker on your network could be trying to get you to visit a fake (and potentially harmful) version of click-that-hood.com. You should not proceed, especially if you have never seen this warning before for this site.

We don’t really advertise HTTPS, but some people somehow use it anyway.

**Comment by specious** [comment]
[Let's Encrypt](https://letsencrypt.org/about/) might be able to supply a free SSL certificate.

**Comment by fnogatz** [comment]
Yeah, the publicly available instance at http://click-that-hood.com/ is currently still hosted by *Code for America*. We will move to another machine of *Code for Germany* and update the DNS record once we upgrade the depencies and make Click that Hood reliably running on recent versions of node.js.

The internet has changed quite a lot in the recent 8 years. With Letsencrypt, SSL certificates are available today free of charge, so we can easily adopt one for `click-that-hood.com`.
